modules/altinnsalary/model/altinnreport.class.php

```diff
@@ -502,6 +525,84 @@ function generateArbeidsforholdArray($salary, $employee, $work_relation, $use_on
       }
       $arbeidsforhold['sisteDatoForStillingsprosentendring'] = strftime('%F', strtotime($last_change_of_work_percentage));
     }
+
+    // select all furlough active in the report period
+    $query_furlough = "SELECT * FROM workrelationfurlough WHERE WorkRelationID = " . $work_relation->WorkRelationID . " AND (('" . $this->period . "-01' BETWEEN Start AND Stop) OR ((YEAR(Start) = YEAR('" . $this->period . "-01') AND MONTH(Start) = MONTH('" . $this->period . "-01')) OR (YEAR(Stop) = YEAR('" . $this->period . "-01') AND MONTH(Stop) = MONTH('" . $this->period . "-01'))))";
```

Comment:
Will between part of the query work for this example:
Sending for 2016-09.
Furlough is from 2016-05-23 and no end date.